site/content/blog/2023-09-23-postman-add-on/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Postman Add-on"
+summary: "Import Postman collections with the new Postman add-on for ZAP."
+type: post
+tags:
+- blog
+- postman
+- gsoc
+- gsoc-2023
+date: "2023-09-23"
+authors:
+    - vitikasoni
+---
+This extension allows us to easily import [Postman collections](https://www.postman.com/collection/) into ZAP, and hence we can then use ZAP’s powerful security testing capabilities to identify potential vulnerabilities in our APIs developed using Postman.
```

Comment:
Start by linking to the help for your extension (which will be available as soon as we publish it) and tell people how to install it (via the Marketplace https://www.zaproxy.org/addons/ )